client/src/pages/courses/view.tsx

```diff
@@ -0,0 +1,150 @@
+import Checkbox from '@/src/shared/components/Checkbox';
+import Dropdown from '@/src/shared/components/Dropdown';
+import Radiobutton from '@/src/shared/components/Radiobutton';
+import SearchBar from '@/src/shared/components/SearchBar/SearchBar';
+import GridIcon from '@/src/shared/icons/GridIcon';
+import ListIcon from '@/src/shared/icons/ListIcon';
+import { LeftSideBar } from '@/src/shared/layouts/RightSideBar/LeftSideContainer';
+import { RightSideBar } from '@/src/shared/layouts/RightSideBar/RightSideBar';
+import Image from 'next/image';
+import { Fragment, useState } from 'react';
+import type { ReactNode } from 'react';
+
```

Comment:
- be specific in naming
![image](https://user-images.githubusercontent.com/73863685/224670010-4e45ba8e-476b-4c60-8478-16cf62c0c596.png)

- this would cause a confusion in the future, because we have _view detail_ as well.
- change this to `courses/list/index.tsx`